pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -102,6 +107,13 @@ effectiveStdenv.mkDerivation rec {
     # - use MakeAvailable instead of the low-level Populate,
     # - use Eigen3::Eigen as the target name (as declared by libeigen/eigen).
     ./0001-eigen-allow-dependency-injection.patch
+    # Incorporate a patch that has landed upstream which exposes new
+    # 'abseil-cpp' libraries & modifies the 're2' CMakeLists to fix a
+    # configuration error that around missing 'gmock' exports.
+    #
+    # This can be dropped once 'onnxruntime' cuts a new release & we update.
+    # https://github.com/microsoft/onnxruntime/commit/b522df0ae477e59f60acbe6c92c8a64eda96cace
+    ./update-re2.patch
```

Comment:
@mweinelt should this comment be updated? i assumed that `onnxruntime` would incorporate changes like this one into minor releases, but `1.18.1` didn't have this fix so i guess it won't land until `1.19.x`?